rust/ql/src/queries/security/CWE-770/UncontrolledAllocationSizeGood.rs

```diff
@@ -0,0 +1,17 @@
+
+const BUFFER_LIMIT: usize = 10 * 1024;
+
+fn allocate_buffer(user_input: String) -> Result<*mut u8, Error> {
+    let size = user_input.parse::<usize>()?;
+    if (size > BUFFER_LIMIT) {
```

Comment:
```suggestion
    if size > BUFFER_LIMIT {
```
Just to follow Rust conventions. More generally it might be a good idea if we run rustfm on our examples in query documentation?